tests/python/unittest/test_extensions.py

```diff
@@ -157,46 +159,10 @@ def test_subgraph():
     # check that result matches one executed by MXNet
     assert_almost_equal(out[0].asnumpy(), out3[0].asnumpy(), rtol=1e-3, atol=1e-3)
 
-@unittest.skipIf(check_platform(), "not all machine types supported")
-@unittest.skipIf(is_cd_run(), "continuous delivery run - ignoring test")
-@unittest.skipIf(default_context().device_type == 'cpu', "ignoring custom_op_gpu test on cpu run")
-def test_custom_op_gpu():
```

Comment:
Why remove the GPU test?

Reply:
its not removed, its refactored. See the "Changes" section of the PR description. Notice that the test was moved to the **test_extensions_gpu.py** file